internal/page/donor/upload_evidence.go

```diff
@@ -97,39 +102,61 @@ func readUploadEvidenceForm(r *http.Request) *uploadEvidenceForm {
 	}
 
 	// upload part
-	part, err = reader.NextPart()
-	if err != nil && err != io.EOF {
-		return &uploadEvidenceForm{Error: err}
-	}
+	var files []File
+	var formLevelError error
+	for {
+		part, err = reader.NextPart()
+		if err == io.EOF {
+			break
```

Comment:
would it not be neater to `return &uploadEvidenceForm{Error: err}` instead for each of these `break`s?

Reply:
Much neater ✅